### PR TITLE
test: adding an endpoint to get withdrawal records for a user

### PR DESCRIPTION
**Pull request by muku314115**
## 1. Summary
Fixes # (issue)

## 2.Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## 5. Checklist

- [ ] Does the Readme need to be updated?

### CODE REVIEW

File: proto/quicksilver/interchainstaking/v1/query.proto [hunk 1]
```diff
@@ -170,6 +175,11 @@ message QueryWithdrawalRecordsResponse {
   cosmos.base.query.v1beta1.PageResponse pagination = 2;
 }
 
+message QueryUserWithdrawalRecordsRequest {
+  string user_address = 1 [(gogoproto.moretags) = "cosmos.AddressString"];
+  cosmos.base.query.v1beta1.PageRequest pagination = 3;
```
Comment (joe-bowman) [inline proto/quicksilver/interchainstaking/v1/query.proto:180]: 3?

Comment (muku314115) [reply proto/quicksilver/interchainstaking/v1/query.proto:180]: Sorry must have missed this, fixed it now

File: proto/quicksilver/interchainquery/v1/query.proto [hunk 2]
```diff
@@ -16,7 +16,8 @@ option go_package = "github.com/ingenuity-build/quicksilver/x/interchainquery/ty
 service QuerySrvr {
   // Params returns the total set of minting parameters.
   rpc Queries(QueryRequestsRequest) returns (QueryRequestsResponse) {
-    option (google.api.http).get = "/quicksilver/interchainquery/v1/queries/{chain_id}";
+    option (google.api.http).get =
```
Comment (joe-bowman) [inline proto/quicksilver/interchainquery/v1/query.proto:19]: @aljo242 - not sure why but something somewhere is causing Mukund and I to format proto in one way, and you another way, so we are constantly changing these lines back and forth between the two. Would be good to understand why this is happening!

**Review by joe-bowman — APPROVED** [review]
lgtm

File: proto/quicksilver/interchainquery/v1/query.proto (hunk 2 above)
Comment (aljo242) [reply proto/quicksilver/interchainquery/v1/query.proto:19]: @joe-bowman @muku314115 what is your `buf` version on your local machine?

Comment (aljo242) [reply proto/quicksilver/interchainquery/v1/query.proto:19]: yeah i would move to latest which is `1.21.0`

I think that is causing the issue

**Review by ajansari95 — APPROVED** [review]
lgtm

**Review by aljo242 — APPROVED** [review]

**Pull request merged**